server/config/google.config.ts

```diff
@@ -2,4 +2,3 @@ require("dotenv").config();
 
 export const clientID = process.env.GOOGLE_CLIENT_ID;
 export const clientSecret = process.env.GOOGLE_CLIENT_SECRET;
-export const callbackURL = process.env.GOOGLE_CALLBACK_URL;
```

Comment:
It's perfectly fine to export config from here, you may put callbackURL back, just assign string value, not env variable. Same for Facebook